hibernate-core/src/main/java/org/hibernate/type/descriptor/jdbc/JdbcTypeDescriptor.java

```diff
@@ -124,6 +124,16 @@ default boolean isNumber() {
 		return false;
 	}
 
+	default boolean isBinary() {
```

Comment:
BLOB?

Reply:
Seems the original intention here is binary (as the counterpart of 'isString()' method), not BLOB. Do you mean adding 'BLOB' to the method body? I checked the type list in 'isString' and found it does not contain CLOB.

Reply:
Though I also do not see `CLOB` nor `NCLOB` listed under `#isString()`.  Guess it depends how these are used.  But if LOBs are not supposed to be included, that should be either better indicated in the method name or at least mentioned in the Javadoc.  

@beikov ?

Reply:
That method only seems to be used for the purpose of deciding how to inline parameter values. I guess `CLOB` and `NCLOB` could be considered as strings as well.

Reply:
Ok, then we should add all 3.  Or document why they are not there.

Let's not hold up the release for that though - https://hibernate.atlassian.net/browse/HHH-14605